packages/assistify-ai/server/methods/SmartiReload.js

```diff
@@ -1,18 +1,19 @@
 /* globals SystemLogger, RocketChat */
 
 import {SmartiProxy, verbs} from '../SmartiProxy';
+import {SmartiAdapter} from '../lib/SmartiAdapter';
 
 Meteor.methods({
 	triggerFullResync() {
 		SystemLogger.info('Full Smarti resync triggered');
 
-		const query = {$or: [{outOfSync: true}, {outOfSync: {$exists: false}}]};
+		const query = {$or: [{outOfSync: true}, {outOfSync: false}, {outOfSync: {$exists: false}}]};
```

Comment:
now this includes pretty much anything, doesn't it? a boolean value can be absent or have the values true or false. Why did you introduce this? Should we not - given the method name - just remove the selection criterion?